CreateDatabase.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE Adoptions (idAdoption int IDENTITY(1, 1) NOT NULL, name varchar(50) NOT NULL, PRIMARY KEY (idAdoption));
-CREATE TABLE Answers (idAnswer int IDENTITY(1, 1) NOT NULL, FK_option int NOT NULL, FK_client int NOT NULL, PRIMARY KEY (idAnswer));
+CREATE TABLE Answers (idAnswer int IDENTITY(1, 1) NOT NULL, [date] datetime DEFAULT GETDATE() NOT NULL, FK_option int NOT NULL, FK_client int NOT NULL, PRIMARY KEY (idAnswer));
```

Comment:
Try to put the default values in the code (is possible to get the db in a different timezone of the application server?)